src/utils/daos_hdlr.c

```diff
@@ -1344,9 +1376,51 @@ fs_copy(struct cmd_args_s *ap,
 		if (rc == 0)
 			num->num_dirs++;
 		break;
+	case S_IFLNK:
+		/* Similar to cp command to dereference one time*/
+		D_ALLOC(symlink_value, DFS_MAX_PATH + 1);
+		if (symlink_value == NULL)
+			D_GOTO(out, rc = -DER_NOMEM);
+		rc = fs_query_symlink(ap, src_file_dfs, src_path, symlink_value);
+		if (rc)
+			goto out;
+		/* To determine parent dir */
+		src_parent_path = dirname((char *)src_path);
+		if (src_parent_path[1] == '\0' && src_parent_path[0] == '/') {
+			/* root directory */
+			D_ASPRINTF(next_src_path, "/%s", symlink_value);
+		} else {
+			D_ASPRINTF(next_src_path, "%s/%s", src_parent_path, symlink_value);
```

Comment:
I think we don't treat root specially here right? it it is root, then path will be like
//symlink_value, i think this should be ok.

Reply:
I just tested it. You are right. No need for the special case for root.